internal/flags/flags.go

```diff
@@ -33,61 +33,68 @@ type Flags struct {
 
 func buildFlags(args *Args) []cli.Flag {
 	return []cli.Flag{
+		// TODO: Maybe we need a better way to get DefaultText from config/default.go
 		&cli.BoolFlag{
 			Name:        "version",
 			Usage:       "print version and build information",
 			Destination: &args.PrintVersion,
 		},
 		&cli.StringFlag{
 			Name:        "root",
-			Usage:       "the directory storing snapshotter working states",
+			Usage:       "directory to save the working state of nydus-snapshotter",
 			Destination: &args.RootDir,
+			DefaultText: "/var/lib/containerd-nydus",
```

Comment:
This can't work and may violate the current theory of how nydus-snapshotter merges configurations from 1. default values, 2. TOML configuration file 3. CLI parameters. The way assigning a default value to CLI parameters will always override values from TOML configuration file

Reply:
@changweige "DefaultText" will not affect the parsing of parameters, it is just a prompt message to let users know the value when no CLI parameters and configuration files are specified. The "Value" field is the real default value of the CLI.

Please refer to: https://cli.urfave.org/v2/examples/flags/#default-values-for-help-output

Reply:
I see. Thanks for your elaboration. Can we use the default values defined in `config/default.go` as the `DefaultText` value? In order to avoid cyclic dependency, we can move function `ParseParameters` to flags package?

Reply:
> I see. Thanks for your elaboration. Can we use the default values defined in `config/default.go` as the `DefaultText` value? In order to avoid cyclic dependency, we can move function `ParseParameters` to flags package?

Good idea, I will try to do it.